src/main/frontend/src/utils/userUtils.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+const host = "localhost:8080";
+
+export const register = async (person) => {
+    const { data } = (await axios.post(`http://${host}/api/v1/registration`, person));
+    return data;
+};
+
+export const login = async (person) => {
+    let returnstmt="";
+    await axios.post(`http://${host}/api/v1/login`, person)
+    .then(response => {
+        returnstmt = response;
+    })
+    .catch(error => {
+        returnstmt = error.response
+    });
```

Comment:
No need for returnstmt. Also `returnstmt` is bad variable name. it is all lowecase and is not descriptive.
```suggestion
    return await axios.post(`http://${host}/api/v1/login`, person)
    .then(response)
    .catch(error => {
        return error.response;
    });
```